src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 14.06px;
+}
+
+.container {
+  height: 100%;
+  line-height: 60px;
+  max-width: 1200px;
+  align-items: center;
+  display: flex;
+  padding: 0 50px;
+  margin: 0 auto;
+  justify-content: space-between;
+}
+
+.logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+.navigation_list {
+  text-transform: uppercase;
+  display: flex;
+  list-style: none;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+  height: 8px;
```

Comment:
The height property for the '.is-active' class is set to 8px, which seems incorrect as it might not display the element as intended. Consider reviewing this value to ensure it aligns with your design requirements.